NorthwindMVC.Infrastructure/Services/UserService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NorthwindMVC.Core;
+using NorthwindMVC.Infrastructure.UnitOfWork;
+
+namespace NorthwindMVC.Infrastructure.Services
+{
+    public class UserService : IUserService
+    {
+        private readonly IUnitOfWork UnitOfWork;
```

Comment:
Naming convention when using private variables should be underscore followed by name of the variable